tests/azure/vm_test.go

```diff
@@ -0,0 +1,119 @@
+package azure_test
+
+import (
+	"fmt"
+	"math/rand"
+	"testing"
+	"time"
+
+	"github.com/NeowayLabs/klb/tests/lib/azure"
+	"github.com/NeowayLabs/klb/tests/lib/azure/fixture"
+)
+
+func genVMName() string {
+	return fmt.Sprintf("klbvmtests%d", rand.Intn(1000))
+}
+
+func testVMCreate(t *testing.T, f fixture.F) {
+
+	vm := genVMName()
+	osType := "Linux"
+	vmSize := "Standard_DS4_v2"
+	username := "core"
+	osDisk := "test.vhd"
+	imageUrn := "OpenLogic:CentOS:7.2:7.2.20161026"
+	customData := ""
+	keyFile := "./testdata/key.pub"
+
+	availSet, vnet, subnet, nic, storAcc := createVMResources(f)
+
+	f.Shell.Run(
+		"./testdata/create_vm.sh",
+		vm,
+		f.ResGroupName,
+		f.Location,
+		osType,
+		vmSize,
+		username,
+		availSet,
+		vnet,
+		subnet,
+		nic,
+		storAcc,
+		osDisk,
+		imageUrn,
+		customData,
+		keyFile,
+	)
+	vms := azure.NewVM(f)
+	vms.AssertExists(t, vm, availSet, vmSize, osType, nic)
+}
+
+func createVMResources(f fixture.F) (string, string, string, string, string) {
```

Comment:
But would still have 5 parameters to return. Do you think named returns would be better in this case?
Well, already created a struct, but can change again for named returns if it is better.